テーブル設計

rohuty opened the pull request:
# What
ER図作成、README記述

# Why
テーブル設計のため

#ER図
https://gyazo.com/d296eb94468ff53f14fae0e916039c51

rohuty committed:
ER図の作成

rohuty committed:
README記述(データベース関連)

masayaomoto commented on README.md:
```diff
@@ -22,3 +22,67 @@ Things you may want to cover:
+| -------- | ------ | ----------- |
+| nickname | string | null: false |
+| email    | string | null: false, unique: true|
+| password | string | null: false, unique: true|
```
```| password | string | null: false, unique: true|```
こちらのカラムはdeviseのgemを使用しての実装になるため、deviseが予め用意している
encrypted_passwordというカラムを用意してあげましょう。

masayaomoto commented on README.md:
```diff
@@ -22,3 +22,67 @@ Things you may want to cover:
+
+### Association
+- belongs_to :user
+- has_one :purchases
```
```has_one :purchases```
こちら単数系の```purchase```としておきましょう！

NaturalRadio replied:
```
has_one :purchases
```
>has_one :purchases
こちら単数系のpurchaseとしておきましょう！

こちら修正されていませんので修正してみましょう！

masayaomoto reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認をお願いいたします！

rohuty committed:
テーブル設計(修正1)

encrypted_passwordカラムを用意。
has_one :purchases→単数形のpurchaseに修正。

rohuty commented:
修正しました。
よろしくお願いします。

NaturalRadio commented on README.md:
```diff
@@ -22,3 +22,67 @@ Things you may want to cover:
+| post_code | string | null: false |
+| prefecture_id | integer | null: false |
+| city | string | null: false |
+| building_name | string | ||
```
`市町村、番地`を登録するカラムは存在しますが、`番地`を登録するカラムが見受けられませんので作成してみましょう！

NaturalRadio reviewed (commented):
レビュー依頼ありがとうございます！
修正点確認させていただきました。
追加でコメントいたしましたので、ご確認お願いします！

rohuty committed:
テーブル設計(修正2)

READMEの60行目のhas_one :purchasesを単数形に変更。

adressテーブルをaddressesテーブルに変更
番地を登録するadressカラムを追加

rohuty commented:
ご指摘ありがとうございます。
再度、修正しました。
確認をお願いします。

NaturalRadio reviewed (commented):
レビュー依頼ありがとうございます！
LGTMとさせていただきます！実装お疲れ様でした！
引き続き実装頑張っていきましょう！
